tekton/src/pipelines/taskdefs/fvt-apps/manage.yml.j2

```diff
@@ -45,8 +54,36 @@
     - name: fvt_test_suite
       value: metrics-api
   runAfter:
-    - fvt-manage-setup
```

Comment:
@alequint fvt-manage-metrics-api needs to run after fvt-manage-setup as it required super user to be setup before running the script. 

Checkout result of pfvtsquadci12 here: https://dashboard.masdev.wiotp.sl.hursley.ibm.com/tests/pfvtsquadci12/testsuite/ibm-mas-manage/metrics-api

![image](https://github.com/ibm-mas/cli/assets/131964221/35eae82e-7787-4e0b-a6fc-c651dcb7b27c)

Reply:
Oh i see.... good catch @unnati-solanki-git, my criteria was the presence of SetupProperties.data as dataInput and I did not see it for metrics-api. But it seems criteria is not that good, and need to review other "Phase 1" tests, if they require the same.

Reply:
From PHASE 1: all the following succeed or we know why failed:
- fvt-manage-monitoring
- fvt-manage-security-audit-logging
- fvt-manage-security-inactive-auth

...so metrics-api is the only one to be moved to other phases :+1: